examples/yoti_example_django/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3
+ARG YOTI_APPLICATION_ID
+ARG YOTI_CLIENT_SDK_ID
+ARG YOTI_KEY_FILE_PATH
+RUN if [ "$YOTI_APPLICATION_ID" = "" ] ; then echo YOTI_APPLICATION_ID not provided; exit 1; else echo YOTI_APPLICATION_ID is $YOTI_APPLICATION_ID ; fi
+RUN if [ "$YOTI_CLIENT_SDK_ID" = "" ] ; then echo YOTI_CLIENT_SDK_ID not provided; exit 1; else echo YOTI_CLIENT_SDK_ID is $YOTI_CLIENT_SDK_ID ; fi
+RUN if [ "$YOTI_KEY_FILE_PATH" = "" ] ; then echo YOTI_KEY_FILE_PATH not provided; exit 1; else echo YOTI_KEY_FILE_PATH is $YOTI_KEY_FILE_PATH ; fi
```

Comment:
Nice one - but this check would work if the user doesn't change the default arg value e.g YOTI_APPLICATION_ID=yourApplicationId.
Maybe check the value is different from the default value 'yourApplicationId' or change the default value to NULL or empty string.

Reply:
Ah yes, I updated the .env file without updating this check, I'll change to check against `yourApplicationId` and the other similar ones. I'll do the same for the flask Dockerfile too